### PR TITLE
add details regarding oauth client creation

### PR DESCRIPTION
**Pull request by championswimmer**
it was confusing without specifying these

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,10 +19,11 @@ It records a spoken request from the connected microphones, sends it to the Goog
 ## Run the sample
 
 - Configure the Google API Console Project to use the [Google Assistant API][google-assistant-api-config].
-- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials].
+- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials]. Make sure to use "other" option and not "Android" when creating Oauth Client
+- Install `click` Python package, `pip install click` (required for the google-oauthlib-tool)
 - Use the [`google-oauthlib-tool`][google-oauthlib-tool] to generate credentials:
 ```
-pip install google-auth-oauthlib[tool]
+pip install google-auth-oauthlib-tool
```
Comment (Fleker) [inline README.md]: Did the previous command, `pip install google-auth-oauthlib[tool]`, fail?

File: README.md [hunk 2]
```diff
@@ -19,10 +19,11 @@ It records a spoken request from the connected microphones, sends it to the Goog
 ## Run the sample
 
 - Configure the Google API Console Project to use the [Google Assistant API][google-assistant-api-config].
-- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials].
+- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials]. Make sure to use "other" option and not "Android" when creating Oauth Client
+- Install `click` Python package, `pip install click` (required for the google-oauthlib-tool)
```
Comment (Fleker) [inline README.md]: Checking `pip freeze`, I do appear to have the package click. But I'd rather not have this instruction here and fix the dependency requirements in the tool itself. I'll check the setup for that.

File: README.md [hunk 3]
```diff
@@ -19,10 +19,11 @@ It records a spoken request from the connected microphones, sends it to the Goog
 ## Run the sample
 
 - Configure the Google API Console Project to use the [Google Assistant API][google-assistant-api-config].
-- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials].
+- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials]. Make sure to use "other" option and not "Android" when creating Oauth Client
```
Comment (Fleker) [inline README.md]: This is true. But we do include instructions on what client to use in the previous step.

File: README.md (hunk 2 above)
Comment (Fleker) [reply README.md]: Looks like package is already required when installing [the tool](https://github.com/GoogleCloudPlatform/google-auth-library-python-oauthlib/blob/master/setup.py#L21)

**Comment by championswimmer** [comment]
I've signed the CLA

File: README.md (hunk 1 above)
Comment (championswimmer) [reply README.md]: Yes it failed

File: README.md (hunk 2 above)
Comment (championswimmer) [reply README.md]: Didn't install `click` automatically for me

File: README.md (hunk 3 above)
Comment (championswimmer) [reply README.md]: There are lot of stackoverflow posts where people have used the "Android" client (given this is **Android** things, common confusion)

Comment (Fleker) [reply README.md]: Have there been a lot of questions about this problem?

I would probably move your additional comment to a separate:

**Note:** line above the Download step to make sure the order is preserved.

File: README.md (hunk 2 above)
Comment (Fleker) [reply README.md]: I don't know why it didn't install, but I wouldn't want to add in an additional step that isn't necessary for most users.

File: README.md (hunk 1 above)
Comment (Fleker) [reply README.md]: Did you follow [the steps here](https://developers.google.com/assistant/sdk/develop/grpc/integrate#authorize_and_authenticate_your_google_account_to_work_with_the_assistant) to create a virtual Python environment before running the pip commands?

File: README.md (hunk 2 above)
Comment (proppy) [reply README.md]: @championswimmer it should install it if you installed it with `pip install google-auth-oauthlib[tool]`

File: README.md (hunk 1 above)
Comment (proppy) [reply README.md]: @championswimmer make sure you updated pip and setuptools with:
```
pip install pip setuptools --upgrade
```

**Pull request closed without merging**